src/runtime/TemporalObject.h

```diff
@@ -65,9 +71,7 @@ class TemporalObject : public Temporal {
         int nanosecond;
         std::string calendar;
 
-        std::string z;
-        std::string offsetString;
-        std::string name;
+        TimeZone* tz;
```

Comment:
Pointer member `tz` should be deallocated correctly if it is dynamically allocated.
What concerns me is any possible memory leak.
(memory leak is one of the hardest defects to fix)